Source/WebCore/platform/graphics/FontSizeAdjust.h

```diff
@@ -44,7 +44,7 @@ struct FontSizeAdjust {
         IcHeight
     };
 
-    std::optional<float> resolve(float computedSize, const FontMetrics& fontMetrics) const
+    std::optional<float> computeValue(float computedSize, const FontMetrics& fontMetrics) const
```

Comment:
I think the original name `resolve` seems right, especially when it is used in a method called `resolveFromFontIfNeeded`.